inrupt/pod-browser
# fix container shrinking on Safari, write out TS

This PR fixes: 
- "Background" container in login page shrinking in Safari

 ### To test:
- Open a new window in Safari and shrink it to about half the height of the screen
- Open the app (login page)
- Maximize the window again and verify that the background occupies the entire height of the container and it is not shrunk to the previous size of the window
